CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [unreleased]
+
+### Fixed
+
+- Error in displaying temperatures at depths in station details. [#96](https://github.com/Oceans-1876/challenger-portal/issues/97)
```

Comment:
Should be #97, you have #96 which matches the PR not the issue

Reply:
ohh Shit :( I didn't see this comment. On it!